### DISCUSSION TITLE
chiaexplorer--- How to find the number of xchs obtained every day before

### DISCUSSION THREAD

**Discussion by Thanatos-JFJ** [Chia-Network/chia-blockchain#10210] [Support]
Now 0.00000000000 1 XCH is displayed in chiaexplorer . How can I find the XCH output of 2021-10-20 @richardkiss @maran @wbendick @sargonas @cross

**Comment by Thanatos-JFJ** [comment]
![1644655565(1)](https://user-images.githubusercontent.com/32827336/153704309-e54d95fc-ac78-4c05-b15c-779cb84500dd.png)

**Comment by loppefaaret** [comment]
by scrolling down the page ? loading more results ? someone is sending you 1 mojo transaction, and there is nothing anyone can do about it - it is an open blockchain.
